AATree/spark/src/spark.adb

```diff
@@ -0,0 +1,156 @@
+with Ada.Text_IO;         use Ada.Text_IO;
```

Comment:
By the way, I think there's a builtin pretty formatter. Just double-checking, is this formatted according to whatever standard there is?